ci-operator/step-registry/ipi/conf/gcp/user-labels-tags/ipi-conf-gcp-user-labels-tags-commands.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/user-labels-tags.yaml.patch"
+
+# user labels
+i=0
+printf '%s' "${USER_LABELS:-}" | while read -r KEY VALUE || [ -n "${KEY}" ]
+do
+  yq-3.3.0 write -i "${PATCH}" "platform.gcp.userLabels[$i].key" "${KEY}"
+  yq-3.3.0 write -i "${PATCH}" "platform.gcp.userLabels[$i].value" "${VALUE}"
+  i=$(( $i + 1))
+done
+
+# user tags
+i=0
+printf '%s' "${USER_TAGS:-}" | while read -r PARENT KEY VALUE || [ -n "${PARENT}" ]
+do
+  yq-3.3.0 write -i "${PATCH}" "platform.gcp.userTags[$i].parentID" "${PARENT}"
```

Comment:
there is `yq-3.3.0` binary in upi-installer image?

Reply:
Sorry, it's my local "yq", let me update to "yq-go", thanks!